### ISSUE TITLE
Can't unlock Signal with the screen lock PIN when a fingerprint is set up

### ISSUE THREAD

**Issue by paride** [signalapp/Signal-Android#9407]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

### Bug description

Fingerprint unlock is handy, but sometimes it just doesn't work, for example when wearing gloves, or if your fingerprint has scratches or marks. Android is smart and it always allows you to unlock the screen with the screen lock PIN (or password) if desired. This isn't the case with Signal: if fingerprint unlock is configured, there is no way to unlock Signal with the screen lock PIN. 

### Steps to reproduce

- Setup everything to unlock Signal with the screen lock fingerprint
- Lock the screen
- Pretend you can't use your fingerprint to unlock
- Happily unlock the phone with the PIN
- See how you can't unlock Signal in any way.

**Expected result:** a PIN unlock option is available.

### Device info

**Device:** Xiaomi Mi A2
**Android version:** 10
**Signal version:** 4.55.8

**Comment by EmberHeartshine** [comment]
Signal doesn't use Android's built-in `BiometricManager.Authenticators` API (or `FingerprintManager` on Android < 9). Signal is the only app that uses biometric auth that doesn't do this. It's bizarre.

**Comment by zakutin** [comment]
Yes, user should be able to choose between the PIN or the fingerprint using in-app screen lock feature, no matter if the fingerprint is set in Android globally.

Since Signal set itself up as a secure, privacy-oriented platform, this can be a drawback for example in some regime countries, where the security services have a better chance to get an information from detained protesters and can physically force them to unblock the app by finger scanning.

Memorized PIN is harder too get.

**Comment by hsghori** [comment]
FWIW I've found signal's fingerprint auth to be unreliable on android (which is a separate issue) which makes this a big problem. I've been locked out of my chats for minutes (if not hours) because the fingerprint auth would not recognize my fingerprint (where my screen lock was working fine) and there isn't an alternative.
